sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -10,42 +10,45 @@ Testing
 There is a TravisCI project configured to execute on every GitHub push, it can
 be viewed at: https://travis-ci.org/bokeh/bokeh.
 
-To run the just the python unit tests, run the command:
+You can run all available tests (python and JS unit tests, as well as example
+and integration tests) from the top level directory by executing:
 
 .. code-block:: sh
 
-    py.test -m 'not (js or examples)' 
+    py.test
 
+.. note::
+    Currently this script does not support Windows.
 
-To run just the examples, run the command:
+To run just the python unit tests, run the command:
 
 .. code-block:: sh
 
-    py.test -m examples 
+    py.test -m 'not (js or examples or integration)'
```

Comment:
We can do that. It's a small bit of grunt work - adding

   import pytest
   pytestmark = pytest.mark.unit

to the top of every python unit test file

but it wouldn't take long.




On 10/20/15 10:02 PM, Bryan Van de Ven wrote:
> In sphinx/source/docs/dev_guide/testing.rst
> <https://github.com/bokeh/bokeh/pull/2991#discussion_r42583567>:
> 
>>  
>>  .. code-block:: sh
>>  
>> -    py.test -m examples 
>> +    py.test -m 'not (js or examples or integration)'
> 
> is there some reason we can't tag the unit tests so that |-m unit| (or
> whatever) is possible? That would be much better.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bokeh/bokeh/pull/2991/files#r42583567>.
>